UI/GameInfoCache.cpp

```diff
@@ -351,7 +347,9 @@ class GameInfoWorkItem : public Task {
 		if (!info_->LoadFromPath(gamePath_)) {
 			return;
 		}
+
 		// In case of a remote file, check if it actually exists before locking.
+		// This is likely not necessary at a
```

Comment:
We could skip for non-remote files, but we call ExistsFast which for remote files won't make any actual network request.

-[Unknown]